app/src/forms/form/formMetadata/routes.js

```diff
@@ -0,0 +1,30 @@
+const routes = require('express').Router();
+const { currentUser, hasFormPermissions } = require('../../auth/middleware/userAccess');
+const rateLimiter = require('../../common/middleware').apiKeyRateLimiter;
+const validateParameter = require('../../common/middleware/validateParameter');
+const P = require('../../common/constants').Permissions;
+
+const controller = require('./controller');
+
+routes.use(currentUser);
+routes.use(rateLimiter);
+
+routes.param('formId', validateParameter.validateFormId);
+
+routes.post('/:formId/formMetadata', hasFormPermissions([P.FORM_READ, P.FORM_UPDATE]), async (req, res, next) => {
```

Comment:
The techdocs say that "Custom Form Metadata allows the owner/designer..." but I think `FORM_UPDATE` is only for the OWNER role. I'm thinking the permissions here are correct, but it's the techdocs that need to change to remove "designer"? Just want to make sure that the right people have access.

Reply:
changed to owner